src/annotations/components/annotationForm/AnnotationForm.test.tsx

```diff
@@ -0,0 +1,226 @@
+import React from 'react'
+import {renderWithRedux} from 'src/mockState'
+import {
+  AnnotationForm,
+  END_TIME_IN_FUTURE_MESSAGE,
+  START_TIME_IN_FUTURE_MESSAGE,
+  TIMES_ARE_SAME_MESSAGE,
+  WRONG_ORDER_MESSAGE,
+} from 'src/annotations/components/annotationForm/AnnotationForm'
+import {fireEvent} from '@testing-library/react'
+import {createDateTimeFormatter} from 'src/utils/datetime/formatters'
+import {REQUIRED_ERROR} from 'src/annotations/components/annotationForm/AnnotationTimeInput'
+
+const setup = (annoType: 'point' | 'range', startTime, endTime) => {
+  const doNothing = () => {}
+
+  return renderWithRedux(
+    <AnnotationForm
+      title="Add"
+      type={annoType}
+      startTime={startTime}
+      endTime={endTime}
+      eventPrefix="testing"
+      onClose={doNothing}
+      onSubmit={doNothing}
+    />
+  )
+}
+
+const setupPoint = time => {
+  return setup('point', time, null)
+}
+
+const setupRange = (start, end) => {
+  return setup('range', start, end)
+}
+
+describe('Annotation Form Component tests', () => {
```

Comment:
```suggestion
describe('Annotation Form Component', () => {
```
nit, but we're describing the form, not the tests :)